orchestration/dap_orchestration/tests/test_pipelines.py

```diff
@@ -1,139 +1,50 @@
-import unittest
```

Comment:
The solids tests that were here previously have been moved to the `test_solids.py` test case. This file exercises the pipeline e2e.